### DISCUSSION TITLE
hdfans现在是不能通过认证了吗?

### DISCUSSION THREAD

**Discussion by sqypan** [jxxghp/MoviePilot#1704] [问题咨询]
有hdfans的用户遇到同我一样的无法认证问题吗? (已过新用户30天期限,已手动重启movie pilot数次)
以下格式无法通过验证,error msg: 未配置站点或未通过用户认证

AUTH_SITE=hdfans
HDFANS_UID=点开用户名后的5位数的数字
HDFANS_PASSKEY=密钥,前后没有跟空格

**Comment by cng10001** [comment]
能认证，只要网页正常访问使用，应该就能认证吧
![image](https://github.com/jxxghp/MoviePilot/assets/115156545/d01898c2-2681-4bd4-aa8b-626381c7d9bc)
